Makefile

```diff
@@ -34,7 +35,6 @@ deb: build
 	-t deb \
 	-v $(VERSION) \
 	--deb-no-default-config-files \
-	--deb-use-file-permissions \
```

Comment:
and there comes the [official answer](https://github.com/jordansissel/fpm/issues/1171#issuecomment-237973680):

> Without the flag, the file is owned by 0 (root). With the flag, the file is owned by the user who owned the file at package-time.

<!---->

> I think I know what's going on. When fpm is staging the files for packaging, it copies them, and if you aren't root, you can't apply ownership. When fpm is run as non-root, this causes the file ownership to be all owned by the user executing fpm.

@dmalan aok to merge?